marklogic-data-hub-central/ui/src/components/modeling/property-table/property-table.test.tsx

```diff
@@ -160,5 +166,43 @@ describe('Entity Modeling Property Table Component', () => {
 
     expect(getByText('newStructure')).toBeInTheDocument();
   });
+
+  test('can edit a basic property', async () => {
+    let entityName = propertyTableEntities[2].entityName;
+    let definitions = propertyTableEntities[2].model.definitions;
+    const { getByText, getByTestId, queryByTestId, getAllByTestId } =  render(
+      <PropertyTable 
+        canReadEntityModel={true}
+        canWriteEntityModel={true}
+        entityName={entityName} 
+        definitions={definitions}
+      />
+    )
+
```

Comment:
Can we add an assertion to ensure entityName cannot be edited in one of theses tests for edit?

Reply:
This is for edit property. entityName is for edit description modal where it can not be edited. Property names can be edited.